generators/php/lists.js

```diff
@@ -356,6 +356,34 @@ Blockly.PHP['lists_getSublist'] = function(block) {
   return [code, Blockly.PHP.ORDER_FUNCTION_CALL];
 };
 
+Blockly.PHP['lists_sort'] = function(block) {
+  // Block for sorting a list.
+  var listCode = Blockly.PHP.valueToCode(block, 'LIST',
+      Blockly.PHP.ORDER_FUNCTION_CALL) || 'array()';
+  var direction = block.getFieldValue('DIRECTION') === '1' ? 1 : -1;
```

Comment:
For Google Charts, we have been using === explicitly because it is more clear especially if we are not sure if there is something crazy going on, and the compiler takes care of doing something simpler if it works out.    I believe there is an ongoing debate in Google about this, actually.